_modules/atmos_flux_inversion.html

```diff
@@ -15,6 +15,8 @@
     <script type="text/javascript" src="../_static/underscore.js"></script>
     <script type="text/javascript" src="../_static/doctools.js"></script>
     <script type="text/javascript" src="../_static/language_data.js"></script>
+    <script async="async" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
+    <script type="text/x-mathjax-config">MathJax.Hub.Config({"extensions": ["tex2jax.js"], "jax": ["input/TeX", "output/HTML-CSS"], "TeX": {"extensions": ["AMSmath.js"]}})</script>
```

Comment:
example where MathJax config is specified twice in the HTML output

Reply:
I see where the `MathJax.Hub.Config(...)` line comes from, but I don't know where the `config=TeX-AMS-MML_HTMLorMML` part comes from.  I'm suspecting that's added by `sphinx.ext..mathjax` itself.  I'll look into how to change that.

Reply:
yeah it seems like that might be the default `mathjax_path` setting
https://github.com/sphinx-doc/sphinx/blob/a721631a541db298f7f9807c9db8433e928cb20b/sphinx/ext/mathjax.py#L98

Reply:
so I guess workaround could be to set `mathjax_path` in your conf to `'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js'`

Reply:
or remove your `mathjax_config`, or just not worry about it

Reply:
Updating to latest Sphinx leaves the duplicate configuration.  I dropped the manual MathJax configuration, which leaves just the first line and seems to display the equations.